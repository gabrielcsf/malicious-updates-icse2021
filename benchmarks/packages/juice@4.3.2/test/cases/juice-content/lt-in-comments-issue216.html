<style>
/* somebody <somebody@gmail.com> */
.content {
  background: #002b36;
  color: #839496;
}
</style>
<div class="content">Hello!</div>