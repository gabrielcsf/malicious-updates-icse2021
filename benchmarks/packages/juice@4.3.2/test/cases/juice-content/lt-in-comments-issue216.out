<div class="content" style="background: #002b36; color: #839496;">Hello!</div>